public/public_images_placeholder-logo_png.bundle.js

```diff
@@ -0,0 +1,13 @@
+(self["webpackChunkpostico_gracehopper"] = self["webpackChunkpostico_gracehopper"] || []).push([["public_images_placeholder-logo_png"],{
+
+/***/ "./public/images/placeholder-logo.png":
+/*!********************************************!*\
+  !*** ./public/images/placeholder-logo.png ***!
+  \********************************************/
+/***/ (() => {
+
```

Comment:
will want this in the .gitignore too

Reply:
Yes I'm abandoning trying to put a locally hosted image in a JSX component for now. We can do it, but it requires changing the webpack config and I don't think it's worth it. There are likely easier ways